spec/models/user_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validations' do
+    context 'when password is too long' do
+      it 'validates password length', :aggregate_failures do
+        password = ('m' * 73).to_s
+        user = build(:user, password: password)
```

Comment:
It's better to assign variables outside of the `it` and then use them

```suggestion
        let(:password) { ('m' * 73).to_s }
        let(:user) { build(:user, password: password) }
      it 'validates password length', :aggregate_failures do
```